Fakebook.Profile/Fakebook.Profile.RestApi/Startup.cs

```diff
@@ -46,19 +47,29 @@ public void ConfigureServices(IServiceCollection services)
                     });
             });
 
-            services.AddAuthentication(
-                JwtBearerDefaults.AuthenticationScheme)
-                .AddJwtBearer(options =>
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+            }).AddJwtBearer(o =>
+            {
+                if (_env.IsDevelopment())
                 {
-                    options.Authority = "https://revature-p3.okta.com/oauth2/default";
-                    options.Audience = "api://default";
+                    o.Authority = "https://localhost:44374";
```

Comment:
we should really just be pulling this from configuration, but this will work.
you need to add the ingress route path for the auth service in the ng repo first, though, or this will break auth in production.